src/material/form-field/form-field.ts

```diff
@@ -217,6 +218,20 @@ export class MatFormField
   // Unique id for the label element.
   readonly _labelId = `mat-form-field-label-${nextUniqueId++}`;
 
```

Comment:
I think that we need some changes in `_syncDescribedByIds` so that the `aria-describedby` continues to point to the hidden hints. It'll also need a unit test.